Jenkinsfile

```diff
@@ -42,12 +47,29 @@ pipeline {
       }
     }
 
+    stage("Create Pull Requests") {
```

Comment:
can we handle the automation piece in a separate PR? I want to discuss if its even necessary a bit further. I think I'd like to have @cmastr try making one this first go round so she can understand how easy it is to do within Github.

Creating it when it is time to release is going to end up with less open PRs that need to be rebased, etc.

Reply:
Removed